streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorNode.java

```diff
@@ -132,9 +132,6 @@ public void process(final K key, final V value) {
         this.value = value;
 
         this.nodeMetrics.metrics.measureLatencyNs(time, processDelegate, nodeMetrics.nodeProcessTimeSensor);
-
-        // record throughput
-        this.nodeMetrics.metrics.recordThroughput(nodeMetrics.nodeThroughputSensor, 1);
```

Comment:
Hmm, I'm not sure that `nodeMetrics.nodeProcessTimeSensor` also measures the throughput? In it's `addLatencySensor` function it does not add any rate metrics right? In addition, this sensor currently is only recorded at source node and processor node, if we remove it, then it is only recording at source node, does that really make sense?